openshift-3.11/group.yml

```diff
@@ -25,81 +25,135 @@ default_image_build_method: imagebuilder
   rhel-7-server-ansible-2.4-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/ansible/2.4/os/
+      baseurl:
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/ansible/2.4/os/
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/ansible/2.4/os/
     content_set:
       default: rhel-7-server-ansible-2.4-rpms
+      ppc64le: rhel-7-server-ansible-2.4-for-power-le-rpms
   rhel-server-rhscl-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhscl/1/os/
+      baseurl:
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhscl/1/os/
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/rhscl/1/os/
     content_set:
       default: rhel-server-rhscl-7-rpms
+      ppc64le: rhel-7-server-for-power-le-rhscl-rpms
   rhel-server-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/os/
+      baseurl:
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/os/
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/os/
       enabled: 1
     content_set:
       default: rhel-7-server-rpms
+      ppc64le: rhel-7-for-power-le-rpms
   rhel-server-extras-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/extras/os/
+      baseurl:
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/extras/os/
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/extras/os/
     content_set:
       default: rhel-7-server-extras-rpms
+      ppc64le: rhel-7-for-power-le-extras-rpms
   rhel-server-optional-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/optional/os/
+      baseurl:
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/optional/os/
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/optional/os/
       enabled: 1
     content_set:
       default: rhel-7-server-optional-rpms
+      ppc64le: rhel-7-for-power-le-optional-rpms
   rhel-server-rh-common-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rh-common/os/
+      baseurl:
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rh-common/os/
+        # FIXME: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/rh-common/os/ does not exist
```

Comment:
@jeffdyoung got any idea what this is supposed to be? it's a new repo in 3.11, not sure what exactly it refers to.

Reply:
@sosiouxme it's actually quite old, in fact deprecated; other repos provide the same or newer versions of the same packages.  rh-common used to be in the 3.10 config as well -- only aos3-installation was using it -- and was completely removed in v3.10.64-1 as part of the multi-arch efforts.  The same should happen here.

Reply:
Thanks @yselkowitz.  Just looked at the repo files @dorzel used for 3.11, can confirm he had success without rhel-common.